.github/workflows/ci.yml

```diff
@@ -27,8 +27,9 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 2 # required by Scrutinizer
-      - run: composer install --no-progress --no-ansi --no-interaction --dev --prefer-dist
-      - run: composer test && composer inspect
+      - run: composer update --no-interaction --dev --prefer-dist
```

Comment:
The repo has no lock file: we need the update command.

Reply:
`install` works exactly the same.

why did you add a progress bar and ANSI colors to the terminal output?

Reply:
In GHA the progress bar does not appear as it is not a terminal. (not a tty)
GHA can display ANSI colors.